src/main/java/uk/gov/justice/probation/courtcaseservice/service/ImmutableCourtCaseService.java

```diff
@@ -184,7 +183,7 @@ public CourtCaseEntity getCaseByCaseIdAndDefendantId(String caseId, String defen
     }
 
     private void validateEntity(String caseId, CourtCaseEntity updatedCase) {
-        checkCourtExists(updatedCase.getCourtCode());
+        checkCourtExists(updatedCase.getHearings().get(0).getCourtCode());
```

Comment:
I think we should validate all the courts here, not just the first. Make a distinct list and validate?